main.py

```diff
@@ -39,33 +39,32 @@ def set_experiment(default_config):
     """ Set up default experiments to replicate the results in the paper:
         "Understanding Disentanglement in Beta-VAE" (https://arxiv.org/pdf/1804.03599.pdf)
     """
-    if default_config['experiment'] == 'custom':
+    if default_config.experiment == 'custom':
```

Comment:
are you sure you can use `.` instead of `[]` for dictionary in pythons ?!? I mean I guess you tested, but I never knew that :O

Reply:
It isn't a dictionary at this stage. It is an Argparser object at this stage.